feature-utils/poly-look/test/react-components/legends.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import {
+  BlockLegend,
+  LineLegend,
+} from "../../src/react-components/legends.jsx";
+
+/**
+ * @jest-environment jsdom
+ */
+const legendValues = [
+  ["No color", "#000000"],
+  ["Gray", "#888888"],
+  ["All colors", "#ffffff"],
+];
+const legendEntries = legendValues.map((l) => {
+  return { description: l[0], color: l[1] };
+});
+const blockLegend = <BlockLegend legend={legendEntries} />;
+const lineLegend = <LineLegend legend={legendEntries} />;
```

Comment:
would it make sense to you to move this to a `beforeEach` inside a global `describe` ? so it's closer to the tests-code and not global

Reply:
Don't see a big difference, since tests are run synchronously. Do you think it would be an improvement?

Reply:
All of them are used now, instead of a single one. Thanks for the comment!